src/scss/loginpage.scss

```diff
@@ -0,0 +1,11 @@
+.loginpage-background {
```

Comment:
Nie jestem pełna że u Ciebie wyświtla sie to zdjęcie, bo nie widzę klasy "loginpage-background__background" w pliku index.php